crates/compiler/gen_llvm/src/llvm/build.rs

```diff
@@ -1083,7 +1083,7 @@ pub fn module_from_builtins<'ctx>(
 
     // Note, running DCE here is faster then waiting until full app DCE.
     let mpm = PassManager::create(());
-    mpm.add_global_dce_pass();
+    // mpm.add_global_dce_pass();
```

Comment:
Should delete this comment before merging.

Reply:
Removed in b7c226212c